### ISSUE TITLE
does RRmpWald need frequency info output three ways?

### ISSUE THREAD

**Issue by mariev** [ABS-dev/PF#21]
Current behavior

```r
ex1 <- RRmpWald(pos ~ tx + cluster(cage), New, compare = c('con', 'vac'))
ex1$freqvec
## pos pos neg pos pos neg neg neg 
##       7      13       2       4 

ex1$xtable
##      [,1] [,2]
## [1,]    7    2
## [2,]   13    4

ex1$multvec
##   vac con Freq
## 1 pos pos    7
## 2 neg pos   13
## 3 pos neg    2
## 4 neg neg    4

```

**Comment by mariev** [comment]
in PF v 9.5.4 `$freqvec` and `$xtable` to be deprecated. Users can create these structures from `$multvec` using the following code:

```r
ex1 <- RRmpWald(pos ~ tx + cluster(cage), New, compare = c('con', 'vac'))
ex1$multvec
#   vac con Freq
# 1 pos pos    7
# 2 neg pos   13
# 3 pos neg    2
# 4 neg neg    4

## as $freqvec
ex1$multvec$Freq
# [1]  7 13  2  4

## as $xtable
ex1$multvec$Freq %>% matrix(., 2, 2, byrow = FALSE)
#      [,1] [,2]
# [1,]    7    2
# [2,]   13    4
```

**Comment by mariev** [comment]
closed via #42
